packages/frontend/app/models/vote.ts

```diff
@@ -0,0 +1,16 @@
+import Model, { attr } from '@ember-data/model';
+
+import Identity from 'emberclear/models/identity';
+import VoteChain from './vote-chain';
+
+export default class Vote extends Model {
```

Comment:
Add an ID field to keep track of votes

Reply:
all models have a default id field of `id`

Reply:
Good to know.